### ISSUE TITLE
HipChat Working?

### ISSUE THREAD

**Issue by mhoffman78** [Hygieia/Hygieia#2195]
Just a few general questions about the HipChat:

- Does the collector work?  I saw some conflicting issues about it working.  The collector is up and running but I am not receiving any data on my dashboard and there are no errors on the console or in the logs

- Does it populate any tables in Mongo?  If so, which ones?

- When configuring HipChat - I am assuming the following configurations:
-ChatOps Server URL - https://companyName.hipchat.com
-Room Name - the room number from the URL https://companyName.hipchat.com/chat/room/**4067442**
-Room Auth Token - token generated from HipChat API - I have tried both View Messages and View Group scopes

Thanks for the help!

**Comment by josefarinna** [comment]
For me isn't working also. I got it working on v2.0.4 but not anymore using master branch.
My configuration is the following:
- ChatOps Server - https://hipchat.mycompanyname.com
- Room Name - 849
- Room Auth Token - the one generated using the room notification tokens.
The command I use is: java -jar .\collectors\misc\chat-ops\target\chat-ops-collector-2.0.5-SNAPSHOT.jar --spring.config.name=chatops --spring.config.location=application.properties

I included an screenshot with the output of the plugin

![image](https://user-images.githubusercontent.com/5385726/40540826-c6c366ba-6019-11e8-8790-a8370e0e7d48.png)
